### ISSUE TITLE
Is stylelint-config-standard compatible with custom syntax like less?

### ISSUE THREAD

**Issue by AoDev** [stylelint/stylelint-config-standard#282]
### What steps are needed to reproduce the bug?

Using config-standard with postcss-less

```
{
  "customSyntax": "postcss-less",
  "extends": ["stylelint-config-standard"]
}
```

with *less* files.

index.less
```less
@import './one.less';
```

one.less

```less
.class1 {
  color: red;
}

.class2 {
  &:extend(.class1);
}

```

### What did you expect to happen?

No issues. It's valid less syntax.

### What actually happened?

I'm getting these errors.

```
less/index.less
 1:9  ✖  Expected "'./one.less'" to be "url('./one.less')"  import-notation

less/one.less
 6:5  ✖  Unexpected unknown function "extend"  function-no-unknown

2 problems (2 errors, 0 warnings)
```

**Comment by jeddy3** [comment]
@AoDev Thanks for the request and for using the template.

The short answer is no. Stylelint's built-in parser, rules, and two official configs are geared towards CSS. Fewer people are using CSS-like languages in favour of vanilla CSS. So we're focused on that.

You can [lint CSS-like languages](https://stylelint.io/user-guide/get-started#linting-css-like-languages-and-css-within-containers) either by:

- extending a community config
- using a custom syntax and configuring the rules yourself

There is a [`stylelint-config-recommended-less`](https://www.npmjs.com/package/stylelint-config-recommended-less) package, but it seems unmaintained. There is no `stylelint-config-standard-less` (yet). The quality and health of a community configs usually depend on the number of people using the CSS-like language. It seems not many people are using Less these days.

If you do publish a `stylelint-config-standard-less` package, please send a PR to [Awesome Stylelint](https://github.com/stylelint/awesome-stylelint) so that other people can find it.

**Comment by AoDev** [comment]
@jeddy3 Thanks for the explanation

* Regarding less: true, I noticed it's diminishing popularity. A pity from my point of view but things are what they are.

* Regarding stylelint + less: it used to work in older versions. Would you be fine with adding a note in the docs that this config is meant to be used with vanilla CSS only?

**Comment by gremo** [comment]
Spent hours with the same problem after upgrading to stylelint 15. Please make a note!
